api/src/api/propagation.ts

```diff
@@ -112,6 +113,8 @@ export class PropagationAPI {
 
   public getBaggage = getBaggage;
 
+  public getActiveBaggage = getActiveBaggage;
```

Comment:
After thinking on this a bit, unlike the trace API, I feel like this needs a corresponding `setCurrentBaggage` to support the scenario where you get whatever is current, add or change that baggage, and then set it again so it propagates.

Reply:
The current context API doesn't allow to just set something active. activate/deactivte is bound to a callback, see `context.with()`.
This protects users from missing deactivate calls or wrong sequence like activate(a) => activate(b) => deactivate(a) => deactivate(b).
So it would be likely something like `withBaggage(baggage, cbFn, thisArg, args)`.

Reply:
Hmm, would that mean that the existing baggage API is incorrect? `setBaggage(context.active(), baggage)`

Reply:
`setBaggage()` is fine, it returns a new `context` instance but it's not active. To make a `context` active one has to call `api.context.with(contextToActivate, fn, thisArg, args)`

Reply:
Aha, I see.